src/ml/inc/base_classifier.h

```diff
@@ -1,20 +1,43 @@
 #pragma once
 
 #include "brainflow_model_params.h"
-
+#include "spdlog/spdlog.h"
 
 class BaseClassifier
 {
 protected:
     struct BrainFlowModelParams params;
-
+    bool skip_logs;
 public:
+    static std::shared_ptr<spdlog::logger> ml_logger;
+    static int set_log_level (int log_level);
+    static int set_log_file (char *log_file);
+
     BaseClassifier (struct BrainFlowModelParams model_params) : params (model_params)
     {
     }
 
     virtual ~BaseClassifier ()
```

Comment:
in destructor of base class and in destructors of subclasses skip_logs variable should be set to true. Its important because spdlog doesnt work if its called from destructors at the program end

Reply:
https://github.com/gabime/spdlog/issues/163#issuecomment-171788629